wakuv2/waku_test.go

```diff
@@ -154,10 +154,35 @@ func TestRelayPeers(t *testing.T) {
 	require.Error(t, err)
 }
 
+func parseNodes(rec []string) []*enode.Node {
+	var ns []*enode.Node
+	for _, r := range rec {
+		var n enode.Node
+		if err := n.UnmarshalText([]byte(r)); err != nil {
+			panic(err)
+		}
+		ns = append(ns, &n)
+	}
+	return ns
+}
+
+// In order to run these tests, you must run an nwaku node
+//
+// Using Docker:
```

Comment:
This integration test looks great, wondering if we want to split it from the unit test, having its own directory and running command.